src/OnnxRuntimeSettings_test.cpp

```
@@ -0,0 +1,32 @@
+    settings.remove("ai/onnxPreferGpu");
+
+    OnnxRuntimeSettings* ort = OnnxRuntimeSettings::instance();
+    ort->loadSettings();
+
+    const bool initial = ort->preferGpu();
+    ort->setPreferGpu(!initial);
+    EXPECT_EQ(ort->preferGpu(), !initial);
+
+    OnnxRuntimeSettings* reloaded = OnnxRuntimeSettings::instance();
+    reloaded->loadSettings();
+    EXPECT_EQ(reloaded->preferGpu(), !initial);
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_PROPERTY is a macro then please configure it.

(unknownMacro)